OmarEspinosaCastro/30-Agosto/Hashset.java

```diff
@@ -0,0 +1,20 @@
+
+package collection;
+
+public class Hashset {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int lines = in.nextInt();
+        String[] left = new String[lines];
+        String[] right = new String[lines];
+        for (int indicePair = 0; indicePair < lines; indicePair++) {
+            left[indicePair] = in.next();
+            right[indicePair] = in.next();
+        }
+        HashSet<String> pairs = new HashSet<>(lines);
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices

Reply:
Added 'list <>' instead of 'ArrayList <>'